src/pages/projects/CreateProject.tsx

```diff
@@ -134,12 +141,14 @@ const CreateProject: FC = () => {
       )
         .then(async () => {
           if (
-            !values.default_instance_storage_pool ||
+            (!values.default_instance_storage_pool &&
+              !values.default_project_network) ||
```

Comment:
```suggestion
            (!values.default_instance_storage_pool && !hasNetwork) ||
```

I suspect we should expect the "none" network value here, not empty value.